docs/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO.py

```diff
@@ -81,24 +81,43 @@
 
 ##############################################################################
 # MET Configuration
-# ---------------------
+# -----------------
+#
+# METplus sets environment variables based on user settings in the METplus
```

Comment:
@lisagoodrich Please replace all of the text in this section with the text "There are no MET tools used in this use case."

Reply:
Done